.github/workflows/codeql.yml

```diff
@@ -0,0 +1,52 @@
+name: "CodeQL"
+
+on:
+  push:
+    branches: [ "master" ]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [ "master" ]
+  schedule:
```

Comment:
why do we need a schedule for it when it already fires on pushes & pull requests ?